priv/schema/rabbit.schema

```diff
@@ -1754,9 +1754,9 @@ fun(Port) when is_integer(Port) ->
     Port > 0 andalso Port < 65535
 end}.
 
-{validator, "byte", "Integer is not 0<i<255",
+{validator, "byte", "Integer is not 0<=i<=255",
```

Comment:
Should this say "Integer *is* in [0, 255]"?

Reply:
Updated!

Reply:
Um, this is the error message, so the negation is expected here. I'll correct.